### PR TITLE
not pass BookRepositoryTest

### PR DESCRIPTION
**Pull request by ArtyomK75**

### CODE REVIEW

**Comment by ArtyomK75** [comment]
image

File: src/test/java/book/store/online/repository/book/BookRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class BookRepositoryTest {
```
Comment (Ivan95kos) [inline src/test/java/book/store/online/repository/book/BookRepositoryTest.java]:
```suggestion
class BookRepositoryIntegrationTest {
```

File: src/test/java/book/store/online/controller/CategoryControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import org.testcontainers.shaded.org.apache.commons.lang3.builder.EqualsBuilder;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class CategoryControllerTest {
```
Comment (Ivan95kos) [inline src/test/java/book/store/online/controller/CategoryControllerTest.java]:
```suggestion
class CategoryControllerIntegrationTest {
```

File: src/test/java/book/store/online/controller/BookControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import org.testcontainers.shaded.org.apache.commons.lang3.builder.EqualsBuilder;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class BookControllerTest {
```
Comment (Ivan95kos) [inline src/test/java/book/store/online/controller/BookControllerTest.java]: also

**Review by Ivan95kos — APPROVED** [review]
Great job, with minor comments

**Pull request merged**